index.html

```diff
@@ -1,12 +1,75 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-<body>
-    <h1>Hello world</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="header">
+      <div class="logo-container"><p class="my-logo">Deborah Fashoro</p></div>
+      <div class="menu-container">
+        <img src="./images/Icon-Menu.png" alt="menu-hamburger-icon" />
+      </div>
+      <nav>
+        <a href="#">Portfolio</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </nav>
+    </header>
+    <main>
+      <!-- Headline Section-->
+      <section class="headline">
+        <div class="header-bg">
+          <div class="about-me-text-container">
+            <div class="primary-text">
+              <p>I'm Deborah.</p><p>Glad to see you!</p>
```

Comment:
- [ ] Please consider using the `h1` tag instead of the `p` tag. This will make your code way mode semantic as this is a header and not a paragraph. This will make the project way cleaner 🧼 😄

Reply:
You're right. Thanks.